youtube_transcript_api/__main__.py

```diff
@@ -39,18 +39,31 @@ def parse_args(args):
         default=False,
         help='If this flag is set the output will be JSON formatted.',
     )
-
+    parser.add_argument(
+        '--http-proxy', dest='http_proxy',
+        default=None, metavar='URL',
```

Comment:
if you use `default=''` here, you won't have to do the conditional assignment in `proxies = {"http": parsed_args.http_proxy or "", "https": parsed_args.https_proxy or ""}`